### ISSUE TITLE
prettyLicense method crashes app

### ISSUE THREAD

**Issue by domdomegg** [nicolas-raoul/apps-android-commons#103]
I think it happens as in 1.12 there is no definition for CC-BY-4.0, as that is the common factor with all my crashes. Will have a look now too (although I am not good with programming :P)

Even though I've added the licenses in, it still shouldn't crash if it can't find a license anyway.

```
USER_COMMENT=Viewing SVG file. From error message may be helpful to know it was licensed under cc-by-4.0
ANDROID_VERSION=6.0.1
APP_VERSION_NAME=1.12
BRAND=samsung
PHONE_MODEL=SM-G920F
CUSTOM_DATA=
STACK_TRACE=java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.String.equals(java.lang.Object)' on a null object reference
	at fr.free.nrw.commons.media.MediaDetailFragment.prettyLicense(MediaDetailFragment.java:342)
	at fr.free.nrw.commons.media.MediaDetailFragment.access$1100(MediaDetailFragment.java:24)
	at fr.free.nrw.commons.media.MediaDetailFragment$4.onPostExecute(MediaDetailFragment.java:212)
	at fr.free.nrw.commons.media.MediaDetailFragment$4.onPostExecute(MediaDetailFragment.java:184)
	at android.os.AsyncTask.finish(AsyncTask.java:651)
	at android.os.AsyncTask.access$500(AsyncTask.java:180)
	at android.os.AsyncTask$InternalHandler.handleMessage(AsyncTask.java:668)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:158)
	at android.app.ActivityThread.main(ActivityThread.java:7224)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1230)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1120)
```

**Comment by nicolas-raoul** [comment]
I remember a similar problem #69 I thought I had fixed it :-/

**Comment by nicolas-raoul** [comment]
https://github.com/nicolas-raoul/apps-android-commons/blob/master/commons/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java#L342

**Comment by domdomegg** [comment]
maybe try:
>        if (licenseKey == null || "".equals(licenseKey) {

(I can't seem to get Android Studio working, so can't test)

**Comment by domdomegg** [comment]
Actually might have found a solution, maybe it's picking up line 341, as if the licenseKey is null there it might break? So put the Log statement in the else of the if statement.

**Comment by domdomegg** [comment]
Actually it seems it is the same error (or stems from the same place) as #106, can't reproduce it with fix #107.
